spread.yaml

```diff
@@ -82,31 +87,17 @@ suites:
     tests/main/:
         summary: Full-system tests for snapd
         prepare: |
-            apt install -y ${SPREAD_PATH}/../snapd_*.deb
-            # Snapshot the state including core.
-            if [ ! -f $SPREAD_PATH/snapd-state.tar.gz ]; then
-                ! snap list | grep core || exit 1
-                snap install test-snapd-tools
-                snap list | grep core
-                snap remove test-snapd-tools
-
-                systemctl stop snapd.service snapd.socket
-                systemctl daemon-reload
-                mounts="$(systemctl list-unit-files | grep '^snap[-.].*\.mount' | cut -f1 -d ' ')"
-                services="$(systemctl list-unit-files | grep '^snap[-.].*\.service' | cut -f1 -d ' ')"
-                for unit in $services $mounts; do
-                    systemctl stop $unit
-                done
-                tar czf $SPREAD_PATH/snapd-state.tar.gz /var/lib/snapd /snap /etc/systemd/system/snap-*core*.mount
-                systemctl daemon-reload # Workaround for http://paste.ubuntu.com/17735820/
-                for unit in $mounts $services; do
-                    systemctl start $unit
-                done
-                systemctl start snapd.socket
+            . $TESTSLIB/prepare.sh
+            if [ "$SPREAD_SYSTEM" = "ubuntu-core-16" ]; then
```

Comment:
"ubuntu-core-16", or "ubuntu-core-16-64"?

I suggest renaming both of the qemu images to have a -64 suffix, so they match Linode, and per note above taking -grub out of both Linode images (we'll need the image: trick for that).